ClientApp/src/app/flights-search-bar/flights-search-bar.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, OnInit, ViewEncapsulation} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {Observable} from 'rxjs';
+import {map, startWith} from 'rxjs/operators';
+
+@Component({
+  selector: 'app-flights-search-bar',
+  templateUrl: './flights-search-bar.component.html',
+  styleUrls: ['./flights-search-bar.component.css']
+})
+export class FlightsSearchBarComponent implements OnInit{
+  ticketsType: number;
+  myControl = new FormControl();
+  options: string[] = ['Minsk', 'Moscow', 'Milan', 'Rome', 'Stockholm', 'Helsinki'];
```

Comment:
Consider moving fake data to separate service which will be replaced with the real one fetching data from the backend later.
This change isn't required for current PR.